src/plugins/intellisense/latex-parsing.tsx

```diff
@@ -223,3 +228,71 @@ export function getPartialFunctionCall(
     }
   }
 }
+
+export function getCorrectableIdentifier(mq: MathQuillField): {
+  ident: string;
+  back: () => void;
+} {
+  let cursor: MQCursor | undefined = mq.__controller.cursor[-1];
+
+  const isInSubscript =
+    mq.__controller.cursor?.parent?._el?.classList.contains("dcg-mq-sub");
```

Comment:
```suggestion
    !!mq.__controller.cursor?.closest(".dcg-mq-sub");
```
Though that changes the behavior in a direction we might not want. (your line allows correcting inside parentheses in a subscript, but mine does not).

Also this brings up an issue: subscripts of `\sum`, `\prod`, `\int`, and `\log` should not be marked as isInSubscript.